content/general/kafka-connector.textile

```diff
@@ -8,15 +8,74 @@ languages:
   - none
 jump_to:
   Help with:
+    - Data from Ably to Kafka#data-ably-kafka
+    - How does Ably Kafka Connector work?#how
+    - Mapping options#mapping-options
+    - Publishing messages with a schema#schema
     - Install the connector#install
     - Configure the connector#configure
     - Tutorials#tutorials
     - See also#see-also
 ---
 
-Use the Ably Kafka Connector to send data from one or more "Kafka topics":https://developer.confluent.io/learn-kafka/apache-kafka/topics/ into Ably "channels":/realtime/channels.
+The Ably Kafka Connector provides a ready-made integration between "Kafka":https://kafka.apache.org/ and Ably. It allows for realtime event distribution from Kafka to web, mobile, and IoT clients, over Ably's feature-rich, multi-protocol pub/sub channels. 
 
-If you need to send data from Ably to Kafka instead, use a "Kafka rule":/general/firehose/kafka-rule.
+**Note:** This connector is verified by Confluent as "Gold":https://www.confluent.io/hub/ably/kafka-connect-ably, following the guidelines set forth by Confluent's "Verified Integrations Program":https://www.confluent.io/verified-integrations-program/.
+
+You can use the Ably Kafka Connector to send data from one or more "Kafka topics":https://developer.confluent.io/learn-kafka/apache-kafka/topics/ into Ably "channels":/realtime/channels. The following diagram illustrates a typical deployment:
+
+<a href="/images/diagrams/ably-kafka-connector-overview.png" target="_blank">
+  <img src="/images/diagrams/ably-kafka-connector-overview.png" style="width: 100%"
+  title="Click to enlarge" alt="Ably Kafka Connector overview">
+</a>
+
+h2(#data-ably-kafka). Data from Ably to Kafka
+
+If you need to send data from Ably to Kafka, use a "Kafka rule":/general/firehose/kafka-rule.
+
+h2(#how). How does the Ably Kafka Connector work?
+
+The Ably Kafka Connector is a sink connector built on top of Kafka Connect. It can be self-hosted or hosted with a third-party provider, the most common being the Confluent Platform. You can download it from either "GitHub":https://github.com/ably/kafka-connect-ably or "Confluent Hub":https://www.confluent.io/hub/ably/kafka-connect-ably and install it into your Kafka Connect workers.
+ 
+The Ably Kafka Connector provides a ready-made integration between Kafka and Ably via your Ably API key. Once installed, you can configure the connector with your Ably API key to enable data from one or more Kafka topics to be published into a single Ably channel or multiple Ably channels. Events are then distributed in realtime to web, mobile, and IoT clients over feature-rich, multi-protocol pub/sub Ably channels optimized for last-mile delivery.
+
+h2(#mapping-options). Mapping options
+
+The Ably Kafka Connector supports two mapping options:
+
+1. Static
```

Comment:
If we number these here, we should probably number the headings below too.

Reply:
As the toolchain adds section numbering automatically, this would result in some odd numbering, so I've just made the list bulleted.